content/operate/rc/databases/configuration/clustering.md

```diff
@@ -98,14 +98,10 @@ You can use the `{...}` pattern to direct related keys to the same hash slot, so
 
 Redis Cloud offers 3 hashing policies, which differ in how hash tags are processed. These hashing policies are not always available.
 
-For accounts created after March 31, 2025, Redis defaults to the [Redis hashing policy](#redis-hashing-policy) **when creating a new database**. For all other accounts, Redis defaults to the [standard hashing policy](#standard-hashing-policy). 
+Redis Cloud defaults to the [Redis hashing policy](#redis-hashing-policy) **when creating a new database**. You can select a different hashing policy during database creation, but you cannot change the hashing policy for an existing database, unless the [custom hashing policy](#custom-hashing-policy) is available.
```

Comment:
@yoavredis - in #2331 you mentioned that you can't change the hashing policy after you create a database, but in the custom hashing policy section it says that you can change it from standard to custom (and vice versa). Have we removed the ability to switch hashing policies?